### PR TITLE
Updated package release notes and tags.

### PR DESCRIPTION
**Pull request by jimmyca15**

### CODE REVIEW

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj [hunk 1]
```diff
@@ -14,6 +14,8 @@
     <Company>Microsoft</Company>
     <PackageLicenseUrl>https://aka.ms/AzureAppConfigurationDotnetCorePublicPreviewEula</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/Azure/AppConfiguration</PackageProjectUrl>
+    <PackageReleaseNotes>For the latest release notes visit https://aka.ms/MicrosoftFeatureManagementReleaseNotes</PackageReleaseNotes>
+    <PackageTags>Microsoft FeatureManagement FeatureFlags AzureAppConfiguration ASPNETCore</PackageTags>
```
Comment (abhilasharora) [inline src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:18]: >ASPNETCore [](start = 80, length = 10)

I'm not sure if casing matters for existing tags, but I've seen "aspnetcore" in lower case for all packages.
Example: [https://www.nuget.org/packages/Microsoft.AspNetCore.Http](https://www.nuget.org/packages/Microsoft.AspNetCore.Http)

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj [hunk 2]
```diff
@@ -14,6 +14,8 @@
     <Company>Microsoft</Company>
     <PackageLicenseUrl>https://aka.ms/AzureAppConfigurationDotnetCorePublicPreviewEula</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/Azure/AppConfiguration</PackageProjectUrl>
+    <PackageReleaseNotes>For the latest release notes visit https://aka.ms/MicrosoftFeatureManagementReleaseNotes</PackageReleaseNotes>
```
Comment (abhilasharora) [inline src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:17]: >For the latest release notes visit https://aka.ms/MicrosoftFeatureManagementReleaseNotes [](start = 25, length = 88)

Should there be a comma between `release notes` and `visit`?

Comment (jimmyca15) [reply src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:17]: Not sure, I'll just change it to "The latest release notes can be found at x"

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj (hunk 1 above)
Comment (jimmyca15) [reply src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:18]: Cool, I'll update it.

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj (hunk 2 above)
Comment (jimmyca15) [reply src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj]: Updated.

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj (hunk 1 above)
Comment (jimmyca15) [reply src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj]: Updated.

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj [hunk 3]
```diff
@@ -14,6 +14,8 @@
     <Company>Microsoft</Company>
     <PackageLicenseUrl>https://aka.ms/AzureAppConfigurationDotnetCorePublicPreviewEula</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/Azure/AppConfiguration</PackageProjectUrl>
+    <PackageReleaseNotes>The latest release notes can be found at https://aka.ms/MicrosoftFeatureManagementReleaseNotes</PackageReleaseNotes>
```
Comment (abhilasharora) [inline src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:17]: The link has the release notes for all versions, not just the latest one. How about `latest release notes` -> `release notes`?

**Review by abhilasharora — APPROVED** [review]

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj (hunk 3 above)
Comment (jimmyca15) [reply src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:17]: Sure.

**Pull request merged**